.ci/integrationTestDownstream.groovy

```diff
@@ -140,6 +140,7 @@ pipeline {
     cleanup {
       script{
         if(integrationTestsGen?.results){
+          log(level: 'INFO', text: "integrationTestsGen -> ${integrationTestsGen.results.toString()}")
```

Comment:
I confess that I don't know what's in `integrationTestsGen` but is there any chance this will accidentally reveal secrets stored in the environment? I don't have any example to point to specifically, but it just came to mind as a question so I thought I'd mention it. :) This is probably fine though.

Reply:
no, it cannot, it is a map with version names and results, it is used to generate an HTML with the results.

![image](https://user-images.githubusercontent.com/5400788/73063056-2e412e80-3e9e-11ea-9568-ebad1c429c68.png)

<img width="887" alt="Screenshot 2020-01-24 at 11 38 51" src="https://user-images.githubusercontent.com/5400788/73063077-37ca9680-3e9e-11ea-998e-c4f94beb31a0.png">

Reply:
Thanks for the explanation. I learned something new. Yay!